keyboards/order/keymaps/default/keymap.c

```diff
@@ -0,0 +1,72 @@
+   * |--------------------------------------------------------------+
+   * |Cap |A  |S  |D  |F  |G  |H  |J  |K  |L  |;  |'  |Ent|PgUp|F15 |
+   * |--------------------------------------------------------------+
+   * |Shi |Z  |X  |C  |V  |B  |N  |M  |,  |.  |/  |Shi|Up |Pgdn|F16 |
+   * |--------------------------------------------------------------+
+   * |Ctr|Win|Alt|cop|pas|Spc|   |   |Alt |FN |Ctr|L  |D  |R   |F17 |
+   * '--------------------------------------------------------------'
+   */
+
+  /* FnLayer
+   * .--------------------------------------------------------------.
+   * | ` ~ |F1 |F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|   |   |
+   * |--------------------------------------------------------------+
+   * | Tab |   |   |   |   |   |   |   |   |   |   |   |   | \ |   |
+   * |--------------------------------------------------------------+
+   * | Caps|   |   |   |   |   |   |   |   |   |   |   |Ent|   |   |
+   * |--------------------------------------------------------------+
+   * | Shi|    |   |   |   |   |   |   |   |   |   |   |Shi|VUP|   |
+   * |--------------------------------------------------------------+
+   * | Ctrl|Win|Alt|cop|pas|   |   |   |   |Alt|FN |Ctrl|  |VDN|   |
+   * '--------------------------------------------------------------'
+   */
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+      KC_GESC ,KC_1     ,KC_2     ,KC_3     ,KC_4     ,KC_5     ,KC_6     ,KC_7     ,KC_8     ,KC_9     ,KC_0     ,KC_MINS  ,KC_EQL   ,KC_BSPC  ,KC_F13 ,
+      KC_TAB  ,KC_Q     ,KC_W     ,KC_E     ,KC_R     ,KC_T     ,KC_Y     ,KC_U     ,KC_I     ,KC_O     ,KC_P     ,KC_LBRC  ,KC_RBRC  ,KC_BSLS  ,KC_F14 ,
+      KC_CAPS ,KC_A     ,KC_S     ,KC_D     ,KC_F     ,KC_G     ,KC_H     ,KC_J     ,KC_K     ,KC_L     ,KC_SCLN  ,KC_QUOT  ,KC_ENT   ,KC_PGUP  ,KC_F15 ,
+      KC_LSFT ,KC_Z     ,KC_X     ,KC_C     ,KC_V     ,KC_B     ,KC_N     ,KC_M     ,KC_COMM  ,KC_DOT   ,KC_SLSH  ,KC_RSFT  ,KC_UP    ,KC_PGDN  ,KC_F16 ,
+      KC_LCTL ,KC_LGUI  ,KC_LALT  ,KC_COPY  ,KC_PASTE ,KC_SPC   ,         ,         ,KC_RALT  ,MO(_FL)  ,KC_RCTL  ,KC_LEFT  ,KC_DOWN  ,KC_RIGHT ,KC_F17),
+
+  [_FL] = LAYOUT(
+
+      KC_GRV  ,KC_F1    ,KC_F2    ,KC_F3    ,KC_F4    ,KC_F5    ,KC_F6    ,KC_F7    ,KC_F8    ,KC_F9    ,KC_F10   ,KC_F11   ,KC_F12   ,KC_TRNS  ,KC_TRNS,
+      KC_TRNS ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS,
+      KC_TRNS ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_ENT   ,KC_TRNS  ,KC_TRNS,
+      KC_LSFT ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_RSFT  ,KC_VOLU  ,KC_TRNS,
+      KC_LCTL ,KC_LGUI  ,KC_LALT  ,KC_COPY  ,KC_PASTE ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_TRNS  ,KC_RALT  ,KC_TRNS  ,KC_RCTL  ,KC_TRNS  ,KC_VOLD  ,KC_TRNS),
+
+  [1] = LAYOUT(
```

Comment:
```suggestion
  [2] = LAYOUT_ortho_5x15(
```

This layer also isn't accessible at all.

Reply:
My encoder switches the layers, I copied that code, I just need to change the maximum number of layers... The code can be found in order.h, is there like a variable with the number of layers? 

uint8_t selected_layer = 0;
void encoder_update_user(uint8_t index, bool clockwise) {
  switch (index) {
    case 0:
      if (!clockwise && selected_layer  < 10) {
        selected_layer ++;
      } else if (clockwise && selected_layer  > 0){
        selected_layer --;
      }
      layer_clear();
      layer_on(selected_layer);
  }
}

Is there a way to change the code without losing your changes?
Thanks a lot!

Reply:
Ahh, okay.

I don't know a variable that holds the number of layers in the firmware, but if anyone could work that out for you I'd think it'd be @drashna.